.gitignore

```diff
@@ -8,4 +8,7 @@ build/
 node_modules
 
 # npm logs
-npm-debug.log*
\ No newline at end of file
+npm-debug.log*
+
+# rollup-plugins-typescript2
+.rpt2_cache
```

Comment:
Nit but there are some odd characters at the end of the list line that show up as garbled characters on the review: 

![image](https://cloud.githubusercontent.com/assets/20712182/26610224/4921a872-4574-11e7-89d5-11c0b6244930.png)